`ecspresso deploy` does not seem to succeed in v1.7.4

`ecspresso deploy`, which was successful in v1.7.3, does not seem to succeed in v1.7.4 with the following error:

```
2021/12/25 18:35:43 my-fargate-service/my-fargate Starting deploy
Service: my-fargate-service
Cluster: my-fargate
TaskDefinition: my-task:77
Deployments:
   PRIMARY my-task:77 desired:3 pending:0 running:3
Events:
2021/12/25 18:35:44 my-fargate-service/my-fargate Registering a new task definition...
2021/12/25 18:35:44 my-fargate-service/my-fargate Task definition is registered my-task:78
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x1b71cc4]

goroutine 1 [running]:
github.com/kayac/ecspresso.(*App).Deploy(0xc000580c00, 0xc000037ab5, 0xc000037ab8, 0xc000037ac0, 0xc000037ac1, 0xc000037ac2, 0x0, 0xc00020ae80, 0xc000037ac4, 0xc000037ac5, ...)
        /home/runner/work/ecspresso/ecspresso/deploy.go:81 +0xa24
main._main(0xc000100058)
        /home/runner/work/ecspresso/ecspresso/cmd/ecspresso/main.go:250 +0x8e6e
main.main()
        /home/runner/work/ecspresso/ecspresso/cmd/ecspresso/main.go:18 +0x25
```

Probably due to #365.

--- a/deploy.go
+++ b/deploy.go
@@ -78,7 +78,7 @@ func (d *App) Deploy(opt DeployOption) error {
 		if err != nil {
 			return errors.Wrap(err, "failed to load service definition")
 		}
-		ds, err := diffServices(sv, newSv, *newSv.ServiceArn, d.config.ServiceDefinitionPath, false)
+		ds, err := diffServices(sv, newSv, "", d.config.ServiceDefinitionPath, false)
 		if err != nil {
 			return errors.Wrap(err, "failed to diff of service definitions")
 		}

